features/layout/blocks/articles/ArticlesByTag2.js

```diff
@@ -15,7 +15,7 @@ import styles from './articlesByTag2.module.scss';
 
 const b = bem(styles);
 
-const ArticlesByTag2 = ({ block, data }) => {
+const ArticlesByTag2 = ({ block, data, imageLoading }) => {
```

Comment:
Wouldn't it be more flexible to pass something like "isVisibleInViewport" and then resolve lazy/eager closer to the image?